zoning/extract/extractor/textract.py

```diff
@@ -0,0 +1,256 @@
+from base import *
+from omegaconf import DictConfig
+import boto3
+import time
+import json
+import os
+import pandas as pd
+import pyarrow as pa
+from tqdm.contrib.concurrent import process_map
+from datasets import load_dataset, Dataset, DatasetDict
+import tqdm
+import numpy as np
+import openai
+
+SCHEMA = pa.schema(
```

Comment:
Do we need to do this in PyArrow or can it just be a dataclass?

Reply:
I deleted it in the latest commit